data_science/palette/notebooks/00 - RGB, LAB, and human colour perception.ipynb

```diff
@@ -0,0 +1,133 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# RGB and CIE\n",
+    "In this project we'll regularly need to calculate distances between pixels in colour space as a proxy for the visual difference between the colours. The simplest way of doing this is to calculate the cosine/euclidian/l2 distance, as written below in maths and python:\n",
```

Comment:
A brief explanation of the variables would be helpful. How about:

> If we have two colours $C_1 = (R_1, G_1, B_1)$ and $C_2 = (R_2, G_2, B_2)$, the _Euclidean distance_ $\Delta C$ is defined as:
>
> $\Delta C := \sqrt{(R_2 - R_1)^2 + (G_2 - G_1)^2 + (B_2 - B_1)^2}$

(Note that LaTeX allows you to drop the braces if they only contain a single term, which makes expressions neater.)

Reply:
sp: “Euclidean” not “Euclidian”.

Pick one name, not three! And maybe a link to the Wikipedia page? https://en.wikipedia.org/wiki/Euclidean_distance